go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/golang-jwt/jwe
+
+go 1.16
```

Comment:
Do you have any particular use case which requires Go 1.16? I was thinking whether, we could set this to 1.18 minimum so we could use generics / type constraints, possibly for the `key` value, as this is one of the biggest problems in the current JWT library.